fundraiser/project/fundraiser-contract/tests/utils/interface/core.rs

```diff
@@ -0,0 +1,65 @@
+use fuels::{
+    contract::call_response::FuelCallResponse,
+    prelude::*,
+    tx::{AssetId, ContractId},
+};
+
+use crate::utils::setup::{Coin, Fundraiser};
+
+pub async fn cancel_campaign(contract: &Fundraiser, id: u64) -> FuelCallResponse<()> {
+    contract.methods().cancel_campaign(id).call().await.unwrap()
+}
+
+pub async fn claim_pledges(contract: &Fundraiser, id: u64) -> FuelCallResponse<()> {
+    contract
+        .methods()
+        .claim_pledges(id)
+        .append_variable_outputs(1)
+        .call()
+        .await
+        .unwrap()
+}
+
+pub async fn create_campaign(
+    contract: &Fundraiser,
+    asset: &ContractId,
+    beneficiary: &Identity,
+    deadline: u64,
+    target_amount: u64,
+) -> FuelCallResponse<()> {
+    contract
+        .methods()
+        .create_campaign(asset.clone(), beneficiary.clone(), deadline, target_amount)
```

Comment:
```suggestion
        .create_campaign(*asset, *beneficiary, deadline, target_amount)
```

Reply:
My usage requires cloning. 
At least, the `asset` can be deref'd because it's a `ContractId` and it compiles. The `Identity` does not implement `Copy` so it does not compile. Given the usage I think it makes sense to leave them as clones.